packages/colord.rb

```diff
@@ -3,35 +3,31 @@
 class Colord < Meson
   description 'colord is a system service that makes it easy to manage, install and generate color profiles to accurately color manage input and output devices.'
   homepage 'https://www.freedesktop.org/software/colord/'
-  version '1.4.5'
+  version '1.4.7'
   license 'GPL-2'
   compatibility 'x86_64 aarch64 armv7l'
-  source_url "https://www.freedesktop.org/software/colord/releases/colord-#{version}.tar.xz"
-  source_sha256 'b774ea443d239f4a2ee1853bd678426e669ddeda413dcb71cea1638c4d6c5e17'
-  binary_compression 'tar.xz'
+  source_url 'https://github.com/hughsie/colord.git'
+  git_hashtag version
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '52e9192284622a986090b7c2b88fd888e0a338e212681f21f3df62dd69670554',
-     armv7l: '52e9192284622a986090b7c2b88fd888e0a338e212681f21f3df62dd69670554',
-     x86_64: '35bb9659086e6c86785609196d8aff86b008584ef139bb22644f71a5909d4ba8'
+    aarch64: '1997467d2cd378d26eb12f5846badd952fe66362f5a56523cf6b2c731c8585e5',
+     armv7l: '1997467d2cd378d26eb12f5846badd952fe66362f5a56523cf6b2c731c8585e5',
+     x86_64: '45ee103244fd062d2f578a6f0b7167c9ea33b5b3762430b7685eb2c51d888f16'
   })
 
   depends_on 'bash_completion'
   depends_on 'dbus'
+  depends_on 'eudev' # R
   depends_on 'glib'
   depends_on 'gobject_introspection'
-  depends_on 'gtk_doc'
   depends_on 'gusb'
   depends_on 'lcms'
   depends_on 'libgudev'
   depends_on 'polkit'
-  depends_on 'vala'
+  depends_on 'vala' => :build
 
-  no_lto
-
-  def self.patch
-    system "sed -i 's,-fstack-protector-strong,-fno-stack-protector,' meson.build"
-  end
-
-  meson_options "-Dsystemd=false -Ddaemon_user=#{USER}"
+  # Building manpages fails due to the supposed lack of docbook-xsl-ns.
+  meson_options "-Dsystemd=false -Ddocs=false -Dman=false -Ddaemon_user=#{USER}"
```

Comment:
`-Dman=true`?

Reply:
In general, it's fine to exclude docs but man pages are good to have and usually have a small size footprint.

Reply:
Can we fix docbook-xsl-ns?

Reply:
I tried and wasn't able to.

Reply:
What's the issue with `docbook-xsl-ns`?

Reply:
```
man/meson.build:6:2: ERROR: Problem encountered: Cannot build man pages without docbook-xsl-ns

A full log can be found at /home/chronos/user/colord/builddir/meson-logs/meson-log.txt
```

extract from log:
```
Running command: /usr/local/bin/xsltproc --nonet http://docbook.sourceforge.net/release/xsl-ns/current/manpages/docbook.xsl
--- stdout ---

--- stderr ---
warning: failed to load external entity "http://docbook.sourceforge.net/release/xsl-ns/current/manpages/docbook.xsl"
cannot parse http://docbook.sourceforge.net/release/xsl-ns/current/manpages/docbook.xsl



man/meson.build:6:2: ERROR: Problem encountered: Cannot build man pages without docbook-xsl-ns
```

Reply:
Why don't we try to build docbook-xsl-ns?  I can see this needed to build other man pages in the future.  See https://sourceforge.net/projects/docbook/files/docbook-xsl-ns/.

Reply:
We currently already package it, to my knowledge-- I can take a look at it later, but for now I'd like to get these security fixes in.

Reply:
Just rebuilt docbook-xsl-nons... Does that help?

Reply:
That could also be a url error. The docbook url needs to be https ...

Reply:
As per https://github.com/docbook/xslt10-stylesheets/pull/259

Reply:
Seeing as docbook is a pain to update because of the lack of upstream releases and also that PR that has been sitting there for a year, I say we say screw it for the colord docs... people can find that stuff online.

Reply:
(And yes I tried to update docbook... and then gave up trying to get it to build from source properly.)